scripts/range.csv

```diff
@@ -0,0 +1,3 @@
+7566651-7566655
+7566656-7566663
```

Comment:
@OlivierBBB looks something like this.

Reply:
So no commas but dashes. Got it 👍

Reply:
a sample range.csv is included in `/scripts`